src/timeMachine/components/view_options/CustomHeatmapOptions.tsx

```diff
@@ -0,0 +1,42 @@
+import React, {FC} from 'react'
+import {getActiveTimeMachine} from 'src/timeMachine/selectors'
+import {useSelector, useDispatch} from 'react-redux'
+import {Form, RangeSlider} from '@influxdata/clockface'
+import {setRadius} from 'src/timeMachine/actions/geoOptionsCreators'
+import {GeoViewProperties} from 'src/client'
+
+enum HeatMapRadiusMinMax {
```

Comment:
```suggestion
enum HeatMapRadiusRange {
```

thoughts on `range` vs `minmax` ?